docs/build-with-powerloom/devnet/Glossary.md

```diff
@@ -0,0 +1,81 @@
+Key terms, concepts, and acronyms used in Powerloom's Devnet
+
+---
+
+### Snapshotter Peers & Nodes
+
+Snapshotter peers are participants in the Powerloom network who capture and verify data snapshots. Snapshotter nodes, run by Snapshotter peers, generate these snapshots at predetermined intervals or epochs. Both are crucial for data accuracy and integrity in the decentralized Powerloom ecosystem.
+
+### Snapshots
+
+Snapshots are compressed archives of blockchain node states at specific block heights. They serve as a historical record of data changes and are taken by Snapshotter Peers. Each snapshot is a report of the observation, specific to state transitions or event logs between two block heights, and is submitted to the Protocol for consensus. Snapshots are essential for ensuring data integrity, enabling analysis, and tracking changes over time within the network.
+
+### Epochs
+
+An Epoch in the context of blockchain typically refers to a range of block heights on the data source blockchain where smart contract applications run, or, simply put, it is a specific period in the blockchain's life.
+
+Each Snapshotter makes observations specific to the state transitions spanning two block heights termed Epochs. Check out our developer docs [here](https://docs.powerloom.io/docs/Protocol/Specifications/Epoch/) to learn more about Epochs.
+
+### RPCs
+
+Remote Procedure Calls (RPCs), allow a program to request a service from another program on a different computer in a network. In the blockchain, an RPC node lets users and applications interact with the blockchain without needing to know network details.
+
+Extracting data from the blockchain state and generating the snapshot can be a complex task. The RPC Helper, defined in utils/rpc.py, has many helper functions to make this process easier. It handles all the retry and caching logic so developers can focus on efficiently building their use cases.
+
+### Prost Chain
+
+The "Prost Chain" refers to the foundational blockchain infrastructure the Powerloom Network utilizes, serving as the anchor chain for storing proofs.
```

Comment:
The anchor chain just does not store proofs. It also coordinates activity between different categories of peers on the network. For eg, it coordinates consensus between snapshotters to finalize snapshots per epoch, it also acts as the distribution point for releasing epochs among so many other functionalities.

Reply:
added the two functionalities, 
- it coordinates consensus between snapshotters to finalize snapshots per epoch, 
- it also acts as the distribution point for releasing epochs